### ISSUE TITLE
Add support to SDK API 30

### ISSUE THREAD

**Issue by CarlosDubon** [blackuy/react-native-twilio-video-webrtc#530]

**Comment by slycoder** [comment]
Are you suggesting changing the default targets?  Does anything bad currently happen when you set the sdk version to 30?

**Comment by CarlosDubon** [comment]
yeah, google play console is request apps with API SDK Target 30, and when i waht compile my app that has integrate twilio, in cosole shows that some anotation of .java files of library are deprecated, anotations like @NonNull @StringDef, i will compile again and update the output
